go/launcher/proxy/quit_handler_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package quithandler
+
+import (
+	"testing"
+
+	"github.com/bazelbuild/rules_webtesting/go/util/bazel"
+	"github.com/bazelbuild/rules_webtesting/go/webtest"
+	"github.com/tebeka/selenium/selenium"
+)
+
+func TestCloseOneWindow(t *testing.T) {
+	testpage, err := bazel.Runfile("go/launcher/proxy/testdata/testpage.html")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	driver, err := webtest.NewWebDriverSession(selenium.Capabilities{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Avoid Fatal so that this defer gets called
```

Comment:
Ahh, I had incorrectly assumed that defer only executed if the function returns normally. Will clean up.